{{cookiecutter.repo_name}}/ci/templates/.travis.yml

```diff
@@ -1,9 +1,12 @@
-language: python
-python: '3.5'
```

Comment:
Specifically set to 3.5 so that it's installed. Travis don't have 3.5 on all the vms and it fails down the road (missing interpreter). Correct me if they fixed this and I'm wrong.